core_backend/app/schemas.py

```diff
@@ -19,6 +20,16 @@ class UserQueryBase(BaseModel):
     model_config = ConfigDict(from_attributes=True)
 
 
+class ResultState(str, Enum):
+    """
+    Enum for Response state
+    """
+
+    FINAL = "final"
```

Comment:
Why do I have FINAL when I don't use it? Idk. 
I thought it would be a nice thing to set in the output check rail. i'll remove it in a future PR if I don't use it.